include/z64actor.h

```diff
@@ -585,13 +586,13 @@ typedef struct TargetContext {
     /* 0x38 */ Actor* arrowPointedActor;
     /* 0x3C */ Actor* targetedActor;
     /* 0x40 */ f32 unk_40;
-    /* 0x44 */ f32 unk_44;
+    /* 0x44 */ f32 reticleRadius;
```

Comment:
out of curiosity, what's the difference between `reticleRadius` and `LockOnReticle.radius`?

Reply:
note that I didnt name these just copied from engineer/MM
But my understanding is that `TargetContext.reticleRadius` is the overall reticle radius that gets updated in Target_Update (later PR), and that value gets copied into individual reticle instances (`LockOnReticle.radius`) as it animates the motion blur animation when it zooms into the actor.

So `LockOnReticle.radius` is all that matters for drawing, while `TargetContext.reticleRadius` is the "master" radius that the reticles read from